zcutil/fetch-params.sh

```diff
@@ -107,13 +107,27 @@ function fetch_params {
 
     if ! [ -f "$output" ]
     then
-        for method in wget ipfs curl failure; do
-            if "fetch_$method" "$filename" "$dlname"; then
-                echo "Download successful!"
-                break
+        for i in 1 2
+        do
+            for method in wget ipfs curl failure; do
+                if "fetch_$method" "${filename}.part.${i}" "${dlname}.part.${i}"; then
```

Comment:
This requires that the files are manually split up into chunks on each fetch remote. I assume this can be handled trivially for the `z.cash` URL, but as-written the IPFS storage bucket would also need to be updated. My understanding of IPFS is that chunking would already occur during fetches (as the fetch is distributed across peers).

I think the script should try IPFS with the original filename, and then try `wget` and `curl` with explicit chunking.

Reply:
ipfs has been broken for this download for quite a while.

Reply:
Please open an issue for fixing the IPFS download, and updating `fetch-params.sh` to target a single file in IPFS.

Reply:
#4102

Reply:
I merged that into #3865.